### PR TITLE
📝 Building from source instructions

### PR DESCRIPTION
**Pull request by grtcdr**
This addresses #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 </h1>
 <h4 align="center">A highly customizable snake clone made in Rust with the <a href="https://github.com/bevyengine/bevy">Bevy engine</a>, named after the Japanese word for snake, <ruby>蛇<rp>(</rp><rt>へび</rt><rp>)</rp></ruby>.</h4>
 
-## Configuration
+## Configuration <a name="configuration"></a>
```
Comment (kroltan) [inline README.md:11]: As far as I know, this isn't needed for GFM, renderers already generate links to headings in `kebab-case`.

File: README.md [hunk 2]
```diff
@@ -136,3 +136,44 @@ In the `data` field is where you make your map. A space means empty space and a
 If you feel like there's something you'd like to add to the game, feel free to make a fork and submit a pull request! I'll try to review it as soon as possible. If there's an issue with how I've structured the code in the project and you feel like there's a better way, however, please make an issue instead. Hebi is a learning project for me to learn Rust and Bevy, and I'd like to implement most of the core changes myself.
 
 If you want to create a custom map generation type, a good starting point would be looking at `src/maps/default.rs`. Once you've got your generator working, submit it with a pull request! The more variety in game maps the better.
+
+
+## Building from source
+
+You've taken interest in the project and would like to work on it or build your own Hebi from source, follow through these steps to get it up and running on your machine.
```
Comment (kroltan) [inline README.md:143]: Feels a bit long-winded (literally!) Maybe split the sentence on the comma, and either:

 - Make the first part a question;
 - Add some preposition to the second part ("Then, follow through [...]", "Now, [...]" etc)

File: README.md [hunk 3]
```diff
@@ -136,3 +136,44 @@ In the `data` field is where you make your map. A space means empty space and a
...
+cargo build
+```
+
+- If you're ready to take your Snake game to the next level and want the best experience
+
+```
+cargo build --release
+```
+
+5. You might want to [configure the game](#configuration) now that you've built it yourself
```
Comment (kroltan) [inline README.md:179]: Might read better if you phrase it the other way around:

> Now that you've built it yourself, you [...]

Comment (kroltan) [reply README.md:179]: Add some sort of punctuation to the instructions. I don't recall exactly, I think ";" is the correct one? Like so:

> 1. List item;
> 2. Another item;
> 3. Last!

**Review by kroltan — CHANGES_REQUESTED** [review]
Looks very good! There are some nitpicks, if you'll allow me.

File: README.md (hunk 1 above)
Comment (grtcdr) [reply README.md:11]: I had no idea, thanks for letting me know :)

File: README.md (hunk 3 above)
Comment (grtcdr) [reply README.md:179]: I usually avoid punctuation when I'm listing things. But if this is not the standard, I'm happy to give it some punctuation bling.

File: README.md (hunk 2 above)
Comment (grtcdr) [reply README.md:143]: > Make the first part a question

It would sound better if it was a question, yes.

File: README.md (hunk 3 above)
Comment (grtcdr) [reply README.md:179]: Yeah, a quick duckduckgo search landed me on what appears to be the standard when it comes to bullet point punctuation, but it's not semicolons, just a period. What do you think?

**Comment by ElnuDev** [comment]
Alright, I got the merged in! I think I kind of screwed up though, I wanted to push some changes I made onto the PR before merging it, and I don't think it worked. :disappointed: (It didn't show up here.) Is that even possible? Anyhow, I made some adjustments and added a section on UPX.

**Comment by grtcdr** [comment]
Oh no :( I was just about to upload the implemented suggestions.

**Comment by grtcdr** [comment]
> Is that even possible

Git works in mysterious ways.

**Pull request merged**